Mail/Commands/NewMessageCommand.swift

```diff
@@ -1,6 +1,6 @@
 /*
  Infomaniak Mail - iOS App
- Copyright (C) 2024 Infomaniak Network SA
+ Copyright (C) 2022 Infomaniak Network SA
```

Comment:
QUID about this, shouldn't we keep the original file creation date ?

Reply:
This is the original creation date for this file. A wrong creation date was probably added later.
If I do:
`git log --follow --format=%ad --date default Mail/Commands/NewMessageCommand.swift | tail -1`
I get:
`Mon May 16 10:34:07 2022 +0200`